src/pocketmine/entity/Effect.php

```diff
@@ -169,8 +169,8 @@ public function getAmplifier(){
 	 *
 	 * @return $this
```

Comment:
Not related to this PR, but I wonder if it is really right to put `$this` here...